bin/wally-tool-chain-install.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
```

Comment:
Please add an SPDX header.

Reply:
I did not realize the SPDX header needs to be added to scripts as well.  There are number of other scripts which need this change.  I'll add them as well.